cmd/atlas/atlas.go

```diff
@@ -124,7 +125,7 @@ func createConfigFromMongoCLIConfig() {
 		return
 	}
 
-	_, _ = fmt.Fprintf(os.Stderr, `AtlasCLI has copied your MongoCLI configuration to: %s
+	_, _ = fmt.Fprintf(os.Stdout, `AtlasCLI has copied your MongoCLI configuration to: %s
```

Comment:
I don't think this one should change, the idea of going to sdterr is so it can be ignored in a script

Reply:
anything that's "informational" usually is directed to stderr

Reply:
thanks for the catch, I probably need to read https://clig.dev/#the-basics again 😅